config/locales/instructions/en.yml

```diff
@@ -17,6 +17,14 @@ en:
           %{resend_code_link}
         fallback_html: If you can't take a phone call right now, you can %{link}
       wrong_number_html: Entered the wrong phone number? %{link}
+    account:
+      reactivate:
+        begin: Let's get started.
+        explanation: 'When you created your account, we gave you a list of words and asked
```

Comment:
seems weird to me to split across two lines like this, should we use the `>` syntax? ditto for the key below

```yaml
explanation: >
  When you created your account ....
```